playbooks/loki.yaml

```
@@ -0,0 +1,44 @@
+# vim:ts=2:sw=2:et:ai:sts=2
+
+# Playbook: Configure loki.
+---
+- name: Configure loki
+  hosts: loki_servers
+  become: true
+  tasks:
+    - name: Install loki
+      ansible.builtin.package:
+        name: loki
+        state: present
+
+    - name: Configure loki
+      ansible.builtin.template:
+        src: files/loki/config.yml.j2
+        dest: /etc/loki/config.yml
+        owner: root
+        group: root
+        mode: '0644'
+
+    - name: Install ferm configuration
+      ansible.builtin.template:
+        src: files/loki/ferm.conf.j2
+        dest: /etc/ferm/ferm.d/svc-loki-{{ item }}.conf
+        owner: root
+        group: adm
+        mode: '0644'
+      loop:
+        - http
+        - grpc
```

Following the convention from other roles, we should have a var to control this. Also, you need to reload ferm after
```suggestion
  notify: _tina_reload_ferm
  when: loki__install_ferm_svc
```